### ISSUE TITLE
Question: what is the inputLimit used for - for Solarflow devices

### ISSUE THREAD

**Issue by reinhard-brandstaedter** [Zendure/developer-device-data-report#26]
I can see there is a inputLimit and a outputLimit for Solarflow devices. While outputLimit controls the output to Home, what exactly is the inputLimit?

**Comment by Zendure** [comment]
Yes, you are correct, the inputLimit property is currently not used
